module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -69,14 +76,24 @@ A vignette on the usage of the `ggplot2` package can be found [**here**](https:/
 | Library/Package                      | Piece of code                                   | What it's called       | What it does                                                                                                                                                                                      |
 |--------------------------------------|-----------------------------------------------------|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
 | `ggplot2`         | [`ggplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggplot)                   | GG Plot               | Begins a plot that is finished by adding layers.        |
+| `ggplot2`               | [`xlab()`; `ylab()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/labs)           | X Axis Labels; Y Axis Labels                    | Modifies the labels on the x axis and on the y axis, respectively        |             
```

Comment:
Since this same link has `ggtitle` in it, you could just put this all on one line.